episodes/anonymization.md

```diff
@@ -333,10 +672,11 @@ Pydicom offers a wide range of capabilities. You can visualize your DICOM data i
 :::::::::::::::::::::::::::::::::::::::: keypoints
 
 - Certain metadata should almost always be removed from DICOM files before sharing
+- Automated tools are available to strip metadata from DICOMs, but manual verification is necessary due to inconsistencies in how fields are utilized
+- Several Python libraries enable access to DICOM metadata
```

Comment:
Would it be an idea to provide examples of such libraries?

Reply:
Well, I actually think these tools are kind of bad and pointless, so I'm not going to point people to them

Reply:
That makes no sense to me. On the one hand people should examine and remove metadata programmatically, and on the other hand it's pointless? What does it mean for a tool to be "bad and pointless"?

Reply:
The automated tools is what I am talking about. Walter's comment then calls them libraries, but my point was that you should use code, and not some silly GUI off the shelf, because there are usually local variations in how people mis0write and overwrite DICOM fields. These off the shelf tools will not get that context, and code will allow you to deal with it. For example in Jerusalem a lot of DICOMs may have patient metadata in fields not intended for it. No automated GUI tool will know these local issues across the world. You need to actually look at your data, and then use code. I hope that clarifies.